finetune/base_models/bert/model.py

```diff
@@ -185,40 +206,53 @@ class RoBERTa(_BaseBert):
     }
     required_files = [
         {
-            "file": os.path.join(FINETUNE_BASE_FOLDER, "model", "bert", "roberta-model-sm-v2.jl"),
+            "file": os.path.join(
+                FINETUNE_BASE_FOLDER, "model", "bert", "roberta-model-sm-v2.jl"
+            ),
             "url": urljoin(ROBERTA_BASE_URL, "roberta-model-sm-v2.jl"),
```

Comment:
just a thought, somewhat unrelated to this PR.  should the actual model file names be hardcoded in here?  seems like we'd be better served if they were in a config or, better yet, the config was itself a database so we could modify without making code changes.

Reply:
Since this is open source I don't think a database is the right play.   Maybe an argument to be made that this should be  a JSON or yaml file or something but "code as config" does afford some niceties that yaml / JSON don't.